### ISSUE TITLE
Input type date with tooltip having issues

### ISSUE THREAD

**Issue by Jujunol** [Dogfalo/materialize#4163]
### Description
When you assign a tooltip to a input[type=date] field and then proceed by clicking or focusing on the date field the tooltip will display. However, even after selecting your date and the modal disappears the tooltip will still display until you hover over it again and then move your mouse away. 

### Example
```
<td>
    <div class="inline">
        <label for="endDate">Ends</label>
        <input type="date" name="endDate" id="endDate" class="tooltipped"
               data-tooltip="This tooltip won't go away if you click on the field...">
    </div>
</td>
....
$('*[type=date]').pickadate({
    format: "yyyy-mm-dd",
    selectMonths: true,
    selectYears: 20
});
```

**Comment by sigmatec** [comment]
This appears to be fixed in 0.99.0 - [JSFiddle](https://jsfiddle.net/wwd96kfa/3/)

**Comment by tomscholz** [comment]
Closed due inactivity. Feel free to reopen it, if it's still necessary.
